packaging/aws-cloudformation/firecamp-autoscalegroup.template

```diff
@@ -90,49 +90,49 @@
     },
     "AWSAMIRegionMap": {
       "AMI": {
-        "AMZNLINUXHVM": "amzn-ami-hvm-2017.03.0.20170417-x86_64-gp2"
+        "AMZNLINUXHVM": "amzn-ami-2017.09.c-amazon-ecs-optimized"
```

Comment:
Thanks for updating the AMI to the latest. While, we could not use ecs-optimized, which already installs the ecs-agent container. The default AWS ECS does not support the external volume driver, see [ecs issue 236 volume driver support](https://github.com/aws/amazon-ecs-agent/issues/236). FireCamp has a simple patch for ecs agent to support firecamp volume plugin. 

Could you please update it to "amzn-ami-hvm-2017.09.1.20171120-x86_64-gp2"? Thanks!

Reply:
I understand, thanks for the explanation! Just updated to the latest amzn-ami